src/NLog/Common/InternalLogger.cs

```diff
@@ -336,6 +280,39 @@ private static void Write([CanBeNull]Exception ex, LogLevel level, string messag
             }
         }
 
+        private static string FormatMessage([CanBeNull]Exception ex, LogLevel level, string message, [CanBeNull]object[] args)
+        {
+            const string TimeStampFormat = "yyyy-MM-dd HH:mm:ss.ffff";
+            const string FieldSeparator = " ";
+
+            var formattedMessage =
+                (args == null) ? message : string.Format(CultureInfo.InvariantCulture, message, args);
+
+            var builder = new StringBuilder(formattedMessage.Length + TimeStampFormat.Length + ex?.ToString()?.Length ?? 0 + 25);
```

Comment:
Resharper says this is  ` ex?.ToString()?.Length ?? (0 + 25)`

So this is wrong ;)